corehq/apps/case_search/templates/case_search/bootstrap3/profile_case_search.html

```diff
@@ -1,82 +0,0 @@
-{% extends 'hqwebapp/bootstrap3/base_section.html' %}
```

Comment:
Is this also the appropriate time to remove `case_search/js/bootstrap3/profile_case_search.js`?

Reply:
Yes, good point. Since the template is being moved/renamed, also seems like a good time to move `case_search/js/bootstrap5/profile_case_search` to `case_search/js/profile_case_search`.

Reply:
Thanks Graham and Jenny! I just unsplit `profile_case_search.js`